README.md

```diff
@@ -16,30 +16,56 @@ The long term goal of the project is to make Yi the editor of choice for the has
 
 We also want to simplify the core Yi package to make it more accessible, splitting some parts into several packages.
 
-Other information (much of it old) is available on the [Haskell wiki][haskellwiki].
+# Getting Started
+### Installing
 
-## Contents
-* [Installing](#installing)
-* [Getting Source](#getting-source)
-* [Reporting Bugs](#reporting-bugs)
-* [Mailing List](#mailing-list)
-* [IRC channel](#irc-channel)
-* [Configuring Yi](#configuring-yi)
-* [Frontend Compatibility](#frontend-compatibility)
-* [Profiling](#profiling)
-* [Reading material](#reading-material)
+If you have cabal installed with ghc>7.8,just run
+    
+    cabal update
+    cabal install yi
+    
+See [this documentation page](http://yi-editor.github.io/pages/installing/)
+for installation instructions(Including Nix).
 
-## Installing
+### Make your own Yi
+Yi with indendation mode:
 
-See [this documentation page](http://yi-editor.github.io/pages/installing/)
-for installation instructions. Hacking instructions if you're using
-the nix package manager are also there.
+    import Yi
```

Comment:
Not sure about having a config file in the README. This stuff would be super-useful in [the documentation](http://yi-editor.github.io/). Thanks for updating the docs btw :)